manager/ui/war/plugins/api-manager/html/api/api-impl.html

```diff
@@ -104,6 +104,19 @@
               </div>
             </div>
 
+            <div style="margin-top: 10px">
+              <input ng-model="updatedApi.disableKeysStrip"
+                     type="checkbox"
+                     id="disable-key-strip"
+                     ng-disabled="isEntityDisabled()">
+              <label for="disable-key-strip"
+                     apiman-i18n-key="disable-key-strip-msg"
+                     style="padding-left: 3px">Disable request keys stripping</label>
+              <a apiman-i18n-key="disable-key-strip-tooltip" type="button" href="javascript:void(0)" class="" data-toggle="popover" title="Stateful Payload Inspection" data-close="true" data-content="If this feature is enabled, the Gateway will not strip the X-API-KEY and apikey query param from the request.  This allows gateway to send the keys to the backend endpoint.">

```

Comment:
There are two spaces: "request.  This allows gateway".
Maybe also change the text to "This allows **the** gateway".

Reply:
Done!